core/src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -30,20 +31,29 @@
 
     private long translogSizeInBytes;
     private int numberOfOperations;
+    private long uncommittedSizeInBytes;
+    private int  uncommittedOperations;
```

Comment:
I wonder if since these are summed in node stats if this should be a long (to reduce the change of overflow)?